css/_darkvariables.scss

```diff
@@ -2,7 +2,7 @@ $blue-500: #0a84d6; /*light to dark in dark theme*/
 $blue-300: #136fac;
 $blue-100: #1d5a83;
 
-$orange-400: #ffc285;
+$orange-400: #c27c36;
 $orange-500: #FFA726;
```

Comment:
but since i changed for the 30% darker orange, what should be orange500 and orange600? Or maybe the notice should be in orange600 and then we can have 10% for orange400, 20% for orange500 and 30% for orange600?